modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -134,19 +134,19 @@ export default {
       }
     }
   },
+  mounted () {
+    this.validate(this.next);
+  },
   methods: {
     validate(value) {
       if (this.field.required && !value.length) {
-        return { message: 'required' };
-      }
-      if (this.limitReached) {
-        this.searchTerm = 'Limit reached!';
-        this.disabled = true;
-      } else {
         this.searchTerm = '';
```

Comment:
Cleaning the search term when removing image, otherwise the text `limit reached` stays